src/mlpack/methods/dbscan/random_point_selection.hpp

```diff
@@ -27,16 +27,24 @@ class RandomPointSelection
   /**
    * Select the next point to use, randomly.
    *
-   * @param unvisited Bitset indicating which points are unvisited.
-   * @param data Unused data.
+   * @param point Unused data.
+   * @param data Dataset to cluster.
    */
   template<typename MatType>
-  static size_t Select(const boost::dynamic_bitset<>& unvisited,
-                       const MatType& /* data */)
+  size_t Select(const size_t /* point */,
+                       const MatType& data /* data */)
   {
-    const size_t max = unvisited.count();
+    // Initialize the length of the unvisited bitset
+    size_t size = data.n_cols; // Get the size of points.
+    if (unvisited.size() != size) 
+    {
+      unvisited.resize(size, true); // Resize & Set bitset to one
+    }
+    
+    // Count the unvisited points and generate n-th index randomly
+    const size_t max = std::count(unvisited.begin(), unvisited.end(), true);
     const size_t index = math::RandInt(max);
-
+    
```

Comment:
Looks like we can remove the extra whitespace here as well.

Reply:
Yes, I will do.